x-pack/heartbeat/monitors/browser/synthexec/enrich.go

```diff
@@ -164,6 +177,9 @@ func (je *journeyEnricher) createSummary(event *beat.Event) error {
 		duration := je.end.Sub(je.start)
 		eventext.MergeEventFields(event, mapstr.M{
 			"monitor": mapstr.M{
+				"check_group": je.checkGroup,
```

Comment:
Don't think it's required here, enrich func should already add these fields even for summary doc.